src/inverseRobot.js

```diff
@@ -8,6 +8,14 @@
 
 function inverseRobot(robot) {
   // write code here
+  const newRobot = {};
+
+  for (const val in robot) {
+    newRobot[robot[val]] = val;
+  }
+
+  return Object.keys(newRobot).length < Object.keys(robot).length
+    ? null : newRobot;
```

Comment:
good, but better to check if key existed in loop
Then you will not iterate through whole array, and return from function when you need it